system-addon/test/unit/lib/LocalizationFeed.test.js

```diff
@@ -87,16 +87,31 @@ describe("Localization Feed", () => {
         too: TEST_STRINGS[DEFAULT_LOCALE].too
       });
     });
-    it("should use all default strings for unknown locale", () => {
-      const locale = "xyz";
+    it("should use multiple fallback strings before default", () => {
+      const locale = "ru";
```

Comment:
I suspect either ```it``` wants to also be a const, or we just want to use both as literals.